odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewDataQualityReport/OverviewDataQualityReportStyles.ts

```diff
@@ -1,122 +1,57 @@
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.BROKEN}`]: {
-          backgroundColor: theme.palette.runStatus.BROKEN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.brokenTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.SKIPPED}`]: {
-          backgroundColor: theme.palette.runStatus.SKIPPED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.skippedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.ABORTED}`]: {
-          backgroundColor: theme.palette.runStatus.ABORTED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.abortedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.UNKNOWN}`]: {
-          backgroundColor: theme.palette.runStatus.UNKNOWN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.unknownTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-      };
-    },
-  });
+export const Container = styled('div')(() => ({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'stretch',
+  alignItems: 'stretch',
+  justifySelf: 'stretch',
+  flexGrow: 1,
+}));
 
-export type StylesType = WithStyles<typeof styles>;
+export const CountLabel = styled(Typography, {
+  shouldForwardProp: propsChecker,
+})<{
+  $dataQualityTestRunStatus: DataQualityTestRunStatusEnum;
+}>(({ theme, $dataQualityTestRunStatus }) => ({
+  color: theme.palette.runStatus[$dataQualityTestRunStatus],
+}));
+
+export const Bar = styled('div', { shouldForwardProp: propsChecker })<{
+  $datasetQualityTestReport?: DataSetTestReport;
+  $dataQualityTestRunStatus: DataQualityTestRunStatusEnum;
+}>(({ theme, $dataQualityTestRunStatus, $datasetQualityTestReport }) => {
+  const succRelation =
+    ($datasetQualityTestReport?.successTotal || 0) /
+    ($datasetQualityTestReport?.total || 1);
+  const otherStatusesAdjustment = 200 / (Math.round(1 - succRelation) + 1);
+  const success = {
+    backgroundColor: theme.palette.runStatus[$dataQualityTestRunStatus],
+    maxWidth: `${(succRelation * 200) / (Math.round(succRelation) + 1)}%`,
+  };
+  const fail = {
+    backgroundColor: theme.palette.runStatus[$dataQualityTestRunStatus],
+    maxWidth: `${
+      ((($datasetQualityTestReport &&
+        $datasetQualityTestReport[
+          `${$dataQualityTestRunStatus.toLowerCase()}Total` as keyof DataSetTestReport
+        ]) ||
+        0) /
+        ($datasetQualityTestReport?.total || 1)) *
+      otherStatusesAdjustment
+    }%`,
+  };
```

Comment:
You duplicate `backgroundColor` property, please, write it in return section.

I think for `success` const we should use another naming, e.g. successTypeBarStyles, because in this constant we describe styles for success bar.
`fail` const includes not only fail type, this one includes a few other types, so, please, use appropriate naming.

And decompose calculations of maxWidth of you `fail` const into a separate function, it's very hard to read.